server/processors/src/main/java/org/hawkular/apm/server/processor/nodedetails/NodeDetailsDeriver.java

```diff
@@ -105,13 +105,7 @@ protected void deriveNodeDetails(Trace trace, List<Node> nodes, List<NodeDetails
                 nd.setCorrelationIds(n.getCorrelationIds());
                 nd.setElapsed(n.getDuration());
```

Comment:
From docs: 
`The elapsed time represents the total time spent within a particular component (including calls to child components)`
maybe better to change it to span finish - span start.